src/main/resources/org/semux/api/swagger/v2.2.0.json

```diff
@@ -804,9 +804,51 @@
                     "format": "int64",
                     "pattern": "^\\d+$"
                 },
+                "gasPrice": {
+                    "type": "string",
+                    "format": "int64",
+                    "pattern": "^\\d+$"
+                },
+                "fee": {
+                    "description": "Return the transaction fee in nano SEM",
+                    "type": "string",
+                    "format": "int64",
+                    "pattern": "^\\d+$"
+                },
                 "code": {
                     "description": "The status of the transaction",
                     "type": "string"
+                },
+                "internalTransactions": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/InternalTransactionType"
+                    }
+                }
+            }
+        },
+        "InternalTransactionType": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "description" : "Transaction type",
+                    "type": "string"
+                },
+                "from": {
+                    "description" : "Sender address",
+                    "type": "string",
+                    "pattern": "^(0x)?[0-9a-fA-F]{40}$"
+                },
+                "to": {
+                    "description" : "Receiver address",
+                    "type": "string",
+                    "pattern": "^(0x)?[0-9a-fA-F]{40}$"
+                },
+                "amount": {
+                    "description" : "Transaction amount",
+                    "type": "string",
+                    "format": "int64",
+                    "pattern": "^\\d+$"
```

Comment:
Another field to indicate a success or failure? I was also thinking about the `depth` and `index`